docs/data/joy/components/table/table.md

```diff
@@ -0,0 +1,272 @@
+---
+product: joy-ui
+title: React Table component
+githubLabel: 'component: table'
+waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/table/
+---
+
+# Table
+
+<p class="description">Tables display sets of data. They can be fully customized.</p>
+
+## Introduction
+
+Joy UI Table lets you use plain HTML structure and turn it into a nice looking table. It has robust configuration that covers all of the basic needs for any projects.
+
+{{"demo": "TableUsage.js", "hideToolbar": true, "bg": "gradient"}}
+
+{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+
+## Basics
```

Comment:
I think this section could use a few sentences—and maybe links to relevant resources—about the various HTML table tags that you can use.

Reply:
Added!